releases_test/releases.test

```diff
@@ -0,0 +1,8 @@
+rebar3 release
+>>>= 1
+rebar3 release --relname app1
+>>>= 0
```

Comment:
Looks good but just to be sure can you add a test to verify the release is built, like:

```
ls _build/default/rel/app1/releases
>>>= 0
```

Reply:
Ah yes, makes a lot of sense!